File: Python3/프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.py
def solution(n, arr1, arr2):
    answer = ["" for _ in range(n)]
    
    for i in range(n):
        binary = str(bin(arr1[i] | arr2[i]))
        binary = binary.lstrip("0b")
    
        while len(binary) < n:
            binary = "0" + binary
            
        for j in range(n):
            answer[i] += "#" if binary[j] == "1" else " "
    
    return answer

File: Swift/프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.swift
func solution(_ n:Int, _ arr1:[Int], _ arr2:[Int]) -> [String] {
    var answer: [String] = Array(repeating: "", count: n)
    
    for i in 0..<n {
        var binary: String = String(arr1[i] | arr2[i], radix: 2)
        
        while binary.count < n {
            binary = "0" + binary
        }
        
        for j in 0..<n {
            answer[i].append(binary[binary.index(binary.startIndex, offsetBy: j)] == "1" ? "#" : " ")
        }
    }
    
    return answer
}